dev/com.ibm.ws.security.token.ltpa_fat/fat/src/com/ibm/ws/security/token/ltpa/fat/LTPAInitializationVectorTests.java

```diff
@@ -0,0 +1,1476 @@
+           // Wait for the LTPA configuration to be ready after the change
+           assertNotNull("Expected LTPA configuration ready message not found in the log.",
+           server1.waitForStringInLog("CWWKS4105I", 5000));
+          updateConfigDynamically(server1, serverConfig);
+
+          // Initial login to simple servlet for form login1
+          String response1 = server1FlClient1.accessProtectedServletWithAuthorizedCredentials(FormLoginClient.PROTECTED_SIMPLE, validUser, validPassword);
+
+        // Get the SSO cookie from the login
+          String server1Cookie = server1FlClient1.getCookieFromLastLogin();
+          assertNotNull("Expected SSO Cookie 1 is missing.", server1Cookie);
+
+
+        // Change the default password  to an invalid value
+          ServerConfiguration server2Config = server2.getServerConfiguration();
+          LTPA ltpa2 = server2Config.getLTPA();
+           setLTPAvalidationKeyPasswordElement(ltpa2, "garbage");
+
+
+          // Attempt to login to the simple servlet on server #2 and assert that the login fails as the IV is set to a different key value for server 2
+          assertTrue("An invalid cookie should result in authorization challenge",server2FlClient1.accessProtectedServletWithInvalidCookie(FormLoginClient.PROTECTED_SIMPLE, server1Cookie));
+    }
+
+    /**
+     * Negative Test Case:
+     * 1. Server #1 and Server #2 contain different primary LTPA keys with different LTPA keys passwords
+     * 2. Use the LTPA Primary key from Server #1 and place it in Server #2, renamed as validation1.key
+     * 3. Replace the `password` property of the validation key with the password from server 1. 
+     * 4. Access a simple servlet with form login using valid credentials on server #1
+     * 5. Authentication should be successful and retrieve the SSO cookie
+     * 6. Set the validation keys of server 2 to an invalid one
+     * 6. Attempt to access the SimpleServlet with form login on server #2 using the same cookie token from Server #1
+     * 7. Authentication fails as the IVs are different, primary keys files are different and the validation key element of server 2
+     * was set to an invalid one.
+     */
+
+    @Mode(TestMode.LITE)
+    @Test
+    @AllowedFFDC({ "javax.crypto.BadPaddingException", "java.lang.IllegalArgumentException" })
+    public void testLTPAValidationKeyUsage_set_passing_validation_key_to_failing_validation_key() throws Exception {
+        // Copy valid ltpa keys to server1. Copy invalid keys to server 2.
+        copyFileToServerResourcesSecurityDir(ALT_VALIDATION_KEY1_PATH, server1);
+         //copying validation1.keys with the same password as server 1 should ensure a succesful login
+        copyFileToServerResourcesSecurityDir(ALT_VALIDATION_KEY1_PATH, server2);
+        copyFileToServerResourcesSecurityDir(ALT_VALIDATION_KEY3_PATH, server2);
+        copyFileToServerResourcesSecurityDir(ALT_VALIDATION_KEY4_PATH, server2);
```

Comment:
why do we copy 3 keys to server2 ?

Reply:
I can see `ALT_VALIDATION_KEY4_PATH` is used at the end of the test but the garbage password part of the test isn't in the test description comment and it doesn't use the global variable when changing the password.